flocker/control/test/test_diffing.py

```diff
@@ -54,6 +56,24 @@ def test_deployment_diffing(self, deployment_a, deployment_b):
             Equals(deployment_b)
         )
 
+    @given(
+        st.lists(deployment_strategy(), min_size=3, max_size=10)
+    )
+    def test_deployment_diffing_composable(self, deployments):
+        """
+        Diffs should compose to create an aggregate diff.
```

Comment:
:question: Haha, It took me a while to get my head around this test. Add a little comment outlining the strategy for my benefit.